# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/alessandro/vsftpd/fix-detection.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

mail to Chris Evans <chris@scary.beasts.org>

--- ./vsf_findlibs.sh.org	2002-10-28 01:12:53.000000000 +0100
+++ ./vsf_findlibs.sh	2003-04-02 15:01:56.000000000 +0200
@@ -1,7 +1,13 @@
 #!/bin/sh
 # Cheesy hacky location of additional link libraries.
 
-locate_library() { [ ! "$1*" = "`echo $1*`" ]; }
+locate_library() { 
+    while [ "$1" ]; do
+	for i in $1* ; do [ -f "$i" ] && return 0; done
+	shift
+    done
+    return 1
+}
 
 if (grep -q "#define VSF_BUILD_TCPWRAPPERS" builddefs.h); then
   echo "-lwrap";
